contracts/controller/src/healthfactor.rs

```diff
@@ -79,6 +80,38 @@ impl Contract {
             self.get_health_threshold()
         }
     }
+
+    pub fn get_health_factor_with_accrued_interest(&self, user_account: AccountId) -> Ratio {
```

Comment:
@dariacherniak we don't want this as separate function, every place where health factor is involved should respect borrow accrued interest